plugins/experimental/tls_bridge/tls_bridge.cc

```diff
@@ -21,7 +21,7 @@
 
 #include "ts/ts.h"
 #include "swoc/TextView.h"
-#include "regex.h"
+#include "tscore/Regex.h"
```

Comment:
I really don't like this - it's cheating once again and creating future problems. If someone is writing a TLS bridge or variant, that plugin isn't permitted to do this, that is grab a file out of "tscore". TLS Bridge was written to be a useful example and this really diminishes that.

Reply:
Yes. This is a symptom of the bigger problem that tsl_bridge is relying upon tscore. This regex that was in there was copied and pasted, and moving things around resulting in the linker seeing duplicate symbols for that regex. Since this relies upon tscore anyway (it was already linking into that before this PR), it made sense to address the errors by removing the copied code. We can address tls_bridge core dependency in a separate PR. I filed a separate ticket for this issue: #10707